<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ti-Sph 3900S</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ap.ico</normaloff>ap.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="nmLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>47</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>nm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goWlButt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>64</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="shutButt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>46</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Shutter</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="spLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>35</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="spEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>65</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setSpButt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>65</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="wlEdit">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>65</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="wlLabel">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>12</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
